### ISSUE TITLE
: cannot use precision Float with weights of type Int32

### ISSUE THREAD

**Issue by zhoutianzi666** [NVIDIA/TensorRT#2076]
## Environment

**TensorRT Version**: 6015
**NVIDIA GPU**: T4
**NVIDIA Driver Version**: 
**CUDA Version**: 10.1
**CUDNN Version**: 
**Operating System**: ubuntu 
**Python Version (if applicable)**: 3.8 
**Tensorflow Version (if applicable)**: 
**PyTorch Version (if applicable)**: 
**Baremetal or Container (if so, version)**: 

## Steps To Reproduce

E0621 05:22:19.918519  5963 helper.h:113] A shape layer can only run in INT32 precision
E0621 05:22:19.918557  5963 helper.h:113] A shape layer can only run in INT32 precision
I0621 05:22:19.918612  5963 engine.cc:206] Run Paddle-TRT Dynamic Shape mode.
E0621 05:22:19.918772  5963 helper.h:113] (Unnamed Layer* 1) [Constant]: cannot use precision Float with weights of type Int32

**Comment by zerollzeng** [comment]
can you provide the TRT verbose log here?

**Comment by nvpohanh** [comment]
Also, TRT 6.0 is too old. Please try TRT 8.4 to see if the issue still exists. Thanks

**Comment by zhoutianzi666** [comment]
> Also, TRT 6.0 is too old. Please try TRT 8.4 to see if the issue still exists. Thanks

this problem has been sloved，thank you
I mistakenly set layer's calculation accuracy to fp32

**Comment by zerollzeng** [comment]
Can we close this?

**Comment by zhoutianzi666** [comment]
> Can we close this?

ok
